projects/angular/src/data/tree-view/tree-node.ts

```diff
@@ -188,6 +199,20 @@ export class ClrTreeNode<T> implements OnInit, OnDestroy {
     this.subscriptions.push(
       this._model.loading$.pipe(debounceTime(0)).subscribe(isLoading => (this.isModelLoading = isLoading))
     );
+
+    this.subscriptions.push(
```

Comment:
Check out the `takeUntil/destroy` pattern [in this PR](https://github.com/vmware-clarity/ng-clarity/pull/42)